cypress/integration/userCanLogIn.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('User can log in', () => {
+  it('successfully', () => {
+    cy.server()
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3002/api/v1/articles',
```

Comment:
please add a beforeEach block for stubbing out articles

Reply:
done